ecosystem/sep-0023.md

```diff
@@ -77,14 +79,20 @@ The following steps transform a binary key into a strkey:
 3.  If we are encoding a multiplexed address, append an 8-byte memo ID
     in network byte order (most significant byte first).
 
-4.  Compute a 16-bit CRC16 checksum of the combined version byte,
-    optional memo ID, and binary key (using polynomial
-    x<sup>16</sup> + x<sup>12</sup> + x<sup>5</sup> + 1). Append the
-    two-byte checksum to the result of the previous step (e.g.,
-    producing a 35-byte quantity for a non-multiplexed ED25519 public
-    key, or 43 byte quantity for a multiplexed one).
+4.  If we are encoding a signed payload, append a 4-byte length in
+    network byte order (most significant byte first) that holds the
+    length of the payload, then append the payload, and finally zero
+    padding of 0 to 3 zero bytes such that the total number of bytes of
+    the payload plus the zero padding is a multiple of four.
 
-3.  Encode the result of the previous step using [RFC4648 base-32
+5.  Compute a 16-bit CRC16 checksum of the combined version byte,
+    optional memo ID, optional payload, and binary key (using
```

Comment:
The unfortunate amendment to this existing sentence makes it sound like memo IDs and payloads can both be part of an address. The earlier steps already establish this, so I don't think there's a big risk of ambiguity here, but maybe we can say `optional memo ID xor payload` as a minimal workaround?

Reply:
Yeah true. I've simplified this section in bf6f538.